native/app/lib/rakuten-books/index.ts

```diff
@@ -30,3 +30,18 @@ export async function searchBookByTitle(title: string, page = 1) {
       return Promise.reject(err.response);
     });
 }
+
+/**
+ * 楽天 Book APIを利用しておすすめの書籍を取得する関数
+ * @returns
+ */
+export async function getRecommendBooks() {
+  const url = `${baseUrl}/${version}?format=${format}&formatVersion=${formatVersion}&applicationId=${applicationId}&sort=sales&hits=${hits}`;
+
+  try {
+    const res: AxiosResponse<ISearchResponse> = await externalInstance.get(url);
+    return res.data;
+  } catch (err) {
+    return Promise.reject(err);
+  }
+}
```

Comment:
ここの返り値って↓みたいのじゃなくても大丈夫そうな感じ?

```suggestion
export async function getRecommendBooks(): Promise<ISearchResponse> {
  const url = `${baseUrl}/${version}?format=${format}&formatVersion=${formatVersion}&applicationId=${applicationId}&sort=sales&hits=${hits}`;

  try {
    const res: AxiosResponse<ISearchResponse> = await externalInstance.get(url);
    return Promise.resolve(res.data);
  } catch (err) {
    return Promise.reject(err);
  }
}
```

Reply:
ありがとう！
awaitすると返り値は Promise.resolve でラップした形で返るから大丈夫🙆‍♂️

Reply:
なるほど!!りょーかい、ありがとう!!